forwards_paper.tex

```diff
@@ -148,71 +133,99 @@ \section*{Introduction}
-corresponds to the vertices in the individual genealogies along the sequence.
+\subsection*{Recording a tree sequence}
+
+A \emph{tree sequence} is an encoding for the sequence of correlated trees 
+that describes the history of a sexual population.
+It is efficient because branches that are shared by adjacent trees are stored once, 
+rather than repeatedly for each tree.
+The topology of a tree sequence is defined via its \emph{nodes} and \emph{edges},
+while information about variants are recorded as \emph{sites} and \emph{mutations}.
+The format for these four tables is given in more detail in the Methods;
+here we explain conceptually,
+using the example of Figure~\ref{fig:example_tree_sequence}.
+
+The \emph{nodes} of a tree sequence 
+correspond to the vertices in the individual genealogies along the sequence,
+and each node refers to a distinct ancestor.
 Since each node represents a specific ancestor, it has a unique ``time'',
 thought of as her birth time, which determines the height of any vertices
-she is associated with. In the example of Figure~\ref{fig:example_tree_sequence}
-we have a total of five nodes. Nodes 0, 1 and 2 occur at time 0 and are our
-samples. Nodes 3 and 4 represent the ancestors of these samples, and were born at
-time 1.5 and 2.5 respectively.
+she is associated with. 
+The example of Figure~\ref{fig:example_tree_sequence} has five nodes:
+nodes 0, 1 and 2 occur at time 0 and are the \emph{samples},
+while nodes 3 and 4 represent those ancestors necessary to record their genealogy,
+who were born at time 1.5 and 2.5 respectively.
 
 \begin{figure}
     \begin{center}
         \includegraphics[width=\textwidth]{example_tree_sequence}
     \end{center}
     \caption{
-        An example tree sequence with three samples over a chromosome of length 10. In the
-        left-most panel we show the tree sequence pictorially in two different ways:
-        in the top are the tree topologies and the bottom shows the spatial extent of the
-        edges that define these topologies. The right-hand panels show the specific encoding
-        of this tree sequence in the four tables (nodes, edges, sites and mutations) defined
-        by \msprime.
+        An example tree sequence with three samples over a chromosome of length 10.
+        The left-hand panel shows the tree sequence pictorially in two different ways:
+        (top) as a sequence of tree topologies 
+        and (bottom) the spatial extent of the edges that define these topologies. 
+        The right-hand panels show the specific encoding
+        of this tree sequence in the four tables (nodes, edges, sites and mutations) 
+        defined by \msprime.
+        \plr{Whoops!  genotype of 1 in the second tree should be `G'!}
         \jk{Some minor alignment issues present here.}
+        \plr{turns out this is no fun to edit in inkscape.  
+            I can re-make to match the other figs
+            but will leave alone for now: it's good!}
```

Comment:
but let me know - would you rather I just re-made the fig in inkscape to match the others?